### PR TITLE
Network metadata updates on chainid.network

### LINKED ISSUE #979: Network metadata updated on chainid.network
Market build won't start due to latest updates on [chainid.network](https://chainid.network/chains.json). Some fields used in the market repo are missing like network for eg.
Update useNetworkMetadata so the build can start again
Merge this also in v4 branch later on
Screenshot 2022-01-06 at 2 46 56 PM

### PR DESCRIPTION
**Pull request by claudiaHash**
Fixes #979  .

Changes proposed in this PR:

- networks metadata parameters fix
- sorting between main and test networks fix

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
There is a typo for the GAIA-X Testnet
Screenshot 2022-01-07 at 11 20 09

**Comment by claudiaHash** [comment]
> There is a typo for the GAIA-X Testnet Screenshot 2022-01-07 at 11 20 09
missed it, thanks!

File: src/utils/web3.ts [hunk 1]
```diff
@@ -49,6 +50,23 @@ export function accountTruncate(account: string): string {
...
+export function getNetworkType(network: EthereumListsChain): string {
+  // HEADS UP! Hack for getting network's type main/test, without using
+  // .network field, which is innexistent on https://chainid.network/chains.json
+  // We hack in mainnet detection for moonriver.
+
+  if (
+    (!network.name.includes('Testnet') &&
+      !network.title?.includes('Testnet') &&
+      network.name !== 'Moonbase Alpha') ||
+    network.name === 'Moonriver'
```
Comment (KatunaNorbert) [inline src/utils/web3.ts:62]: I don't think we need this check for 'Moonriver', it should label it as `mainnet` without this check

**Review by bogdanfazakas — APPROVED** [review]
LGTM now, we can ignore the preview deployment failing since it's configured to run with nextjs not gatsby anymore

**Review by KatunaNorbert — APPROVED** [review]

File: src/utils/web3.ts [hunk 2]
```diff
@@ -49,12 +50,30 @@ export function accountTruncate(account: string): string {
...
+  }
+}
+
 export function getNetworkDisplayName(
   data: EthereumListsChain,
   networkId: number
 ): string {
   let displayName
 
+  console.log('DATA_NETW: ', data, networkId)
```
Comment (kremalicious) [inline src/utils/web3.ts:75]: missing console.log

Comment (claudiaHash) [reply src/utils/web3.ts]: fixed, thanks!

**Pull request merged**